templates/tfengine/recipes/org/project.yaml

```diff
@@ -12,32 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+{{$output_path := get . "OUTPUT_PATH"}}
+{{if not $output_path}}
```

Comment:
Could this be the default value of `get`?

Reply:
Good idea